src/aap_eda/api/serializers/project.py

```diff
@@ -176,6 +192,14 @@ class Meta:
             *read_only_fields,
         ]
 
+    def get_proxy(self, obj: models.Project) -> str:
+        if not obj.proxy:
+            return ""
+        url = obj.proxy
+        if isinstance(url, SecretValue):
```

Comment:
In the migration you are not encrypting existing values. I think it is because we assume there will not be unencrypted values because the field was implemented a few days ago. So, we always expect an encrypted value and this conditional should not be necessary.  You follow the same logic at the import logic. 

Otherwise I think dealing with a field that might encrypted or not would be inconsistent, so it would be better to encrypt the existing ones in the migration.

Reply:
nobody yet started to use this setting. Now directly amend to the original proxy field. No need an extra migration.

Reply:
Agree, what I'm saying is that you don't need to check if the field is a SecretValue, because it should be a SecretValue always.

Reply:
We have to check the type during the create process because it is still the plain text at runtime, haven't been updated to the encrypted value.

Reply:
ok